scripts/kvmd-bootconfig

```diff
@@ -48,6 +48,13 @@ rw
 # ========== First boot configuration ==========
 
 if [ -n "$FIRSTBOOT$FIRST_BOOT" ]; then
+    ### stop regular kvmd-oled service and show first time setup status in oled
+    systemctl stop kvmd-oled
```

Comment:
Why we need to stop it here?

Reply:
If you don't temporarily stop kvmd-oled service, the message will get clobbered.

Reply:
Got it